mava/systems/tf/mad4pg/networks.py

```diff
@@ -175,3 +181,154 @@ def make_default_networks(
         "critics": critic_networks,
         "observations": observation_networks,
     }
+
+
+def make_1d_conv_networks(
```

Comment:
Again, not sure if we should have this in Mava?

Reply:
See above. I am happy to remove it or leave it.